kickthemout.py

```diff
@@ -151,7 +151,7 @@ def to_CIDR_notation(bytes_network, bytes_netmask):
         return net
 
     iface_routes = [route for route in scapy.config.conf.route.routes if route[3] == scapy.config.conf.iface and route[1] != 0xFFFFFFFF]
-    network, netmask, _, interface, address = max(iface_routes, key=lambda item:item[1])
+    network, netmask, _, interface, address, _ = max(iface_routes, key=lambda item:item[1])
```

Comment:
Thank you for your contribution! Let's make this compatible with both old and new versions of scapy:
```
try:
  network, netmask, _, interface, address = max(iface_routes, key=lambda item:item[1])
except:
  network, netmask, _, interface, address, _ = max(iface_routes, key=lambda item:item[1])
```